lib/api/controllers/documentController.js

```diff
@@ -162,7 +162,7 @@ class DocumentController {
         modifiedRequest = newRequest;
 
         request.input.args.state = 'pending';
```

Comment:
It appears to be useless, and for a long time to boot.

Removed this assignation from all of this controller's methods.